Bottom: c2259eda1f85597f19254e2a09e2c111c5274106
Top:    de7bf3b50e5ed8374cae1a9ccb7de90565db3f8e
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-06-28 09:41:16 +0800

Refresh of virtio-net-serialize-tx

---

diff --git a/drivers/net/virtio_net.c b/drivers/net/virtio_net.c
index 1f8c15c..a18f859 100644
--- a/drivers/net/virtio_net.c
+++ b/drivers/net/virtio_net.c
@@ -1802,6 +1802,7 @@ static void virtnet_freeze_down(struct virtio_device *vdev)
 	flush_work(&vi->config_work);
 
 	netif_device_detach(vi->dev);
+	netif_tx_disable(vi->dev);
 	cancel_delayed_work_sync(&vi->refill);
 
 	if (netif_running(vi->dev)) {
